unittest/test_boxqp.cpp

```diff
@@ -31,7 +31,7 @@ void test_unconstrained_qp_with_identity_hessian() {
   Eigen::VectorXd gradient = Eigen::VectorXd::Random(nx);
   Eigen::VectorXd lb = -std::numeric_limits<double>::infinity() * Eigen::VectorXd::Ones(nx);
   Eigen::VectorXd ub = std::numeric_limits<double>::infinity() * Eigen::VectorXd::Ones(nx);
-  Eigen::VectorXd xinit(nx);
+  Eigen::VectorXd xinit = Eigen::VectorXd::Random(nx);
```

Comment:
It would have been good to highlight this change in the PR summary... looking over 96 files takes a while and this is quite an important fix to the unit test.

Reply:
The main changes are described in the description, I believe.

This modification provides a proper method to initialize the vector. Note that before we're not initializing the values.

Reply:
Yes, I understand the changes well. The summary should serve as a summary though in case someone does not read every PR in full length (it's a lot of files...). This is an important catch-and-fix of a bug of a unit test that is likely to not have tested something useful before, and now works well. So it would be good to include these notes in a summary :-).

Reply:
I didn't think it was significant to point out this specific modification.

I will try to do my best in the next time :)

It would be great if you can try these modifications yourself. Remember that you need to match the same compilation flag (i.e. `-march=native`) in each dependency of Crocoddyl that dependents on Eigen.